### PR TITLE
README: add install instruction for Plasma 5 users

### LINKED ISSUE #13: AUR package should use the commit with the latest tag when building
Currently, the AUR package of this effect tries to build the effect from the latest commit. Since the last few commits require KDE6, it fails to build on KDE 5.27. To fix this, one can create a stable branch and merge every release with that branch, or treat the commit with the latest tag as the latest release. 

Using this command before attempting to generate CMake build files should be enough:
```git checkout $(git rev-list --tags --max-count=1)```

Thank you for creating this effect. Looks gorgeous.

### PR DESCRIPTION
**Pull request by zjuyk**
closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,15 @@ Mouse navigation:
...
+For users who are using KF6/Plasma 6
 ```sh
 yay -S kwin-effects-cube-git
 ```
-
+For users who are using Plasma 5
+```sh
+yay -S kwin-effects-cube
+```
+Hint: Please reload kwin after installation to make this effect work.
```
Comment (zzag) [inline README.md]: This should be unnecessary

Comment (zjuyk) [reply README.md]: But on my Plasma 5, it's a must step. We should relaod kwin using `kwin_x11 --replace` or `kwin_wayland --replace` to activate this effect, refer https://github.com/zzag/kwin-effects-cube/issues/6#issuecomment-1328268440

Comment (zzag) [reply README.md]: That doesn't make sense. Meta+C should work after the effect is enabled in system settings. If not, perhaps there's a shortcut conflict

Comment (zjuyk) [reply README.md]: Please wait some time, I'll ask some friends to test both on Xorg and Wayland.

Comment (zjuyk) [reply README.md]: There could be something wrong on my Plasma config. My friends announce that it works without kwin reload.

**Comment by zzag** [comment]
Thanks!

**Pull request merged**